crates/host/src/lib.rs

```diff
@@ -27,39 +33,99 @@ pub struct ExecutionArg {
     pub tree_db: Option<Rc<RefCell<dyn TreeDB>>>,
 }
 
+impl ContextOutput for ExecutionArg {
+    fn get_context_outputs(&self) -> Arc<Mutex<Vec<u64>>> {
+        self.context_outputs.clone()
+    }
+}
+
+impl From<Sequence> for ExecutionArg {
```

Comment:
delete `Sequence` since `dry-run` daemon mode is removed.

Reply:
removed